package.json

```diff
@@ -55,6 +55,7 @@
     "request": "2.79.0",
     "semver": "5.3.0",
     "socket.io": "1.7.2",
+    "socket.io-client": "1.7.2",
```

Comment:
Under Node v4 / npm v2 (I don't know when they changed that), packages are not all at the root level, so this will cause to have one directory in `node_modules/` and one in `node_modules/socket.io/node_modules/`.

But more generally, any reason why not putting the dep in `devDependencies` if we (we being `thelounge` package, i.e. excluding what `socket.io` does) don't require it directly?
Do you know if there is a way to not need it at all, like `require("socket.io/client")` or something like that in your test? That would be optimal as it would use the same piece of code we actually serve.